src/main/java/org/ssu/edu/teachua/ui/components/modal/AddLocationComponent.java

```diff
@@ -55,57 +56,66 @@ public AddLocationComponent(WebDriver driver) {
         super(driver);
     }
 
+    @Step("Close 'Додати локацію' window")
     public AddCenterMainInfoComponent pressCloseAddLocationWindow() {
         waitForElementToBeClickable(closeAddLocationWindowButton).click();
         return new AddCenterMainInfoComponent(driver);
     }
 
+    @Step("Enter location name")
     public AddLocationComponent enterLocationName(String locationName) {
         waitForElementToBeClickable(this.locationName).click();
         this.locationName.clear();
         this.locationName.sendKeys(locationName);
         return this;
     }
 
+    @Step("Select a city from a drop-down list")
```

Comment:
add parameters to step  `@Step("Select a city {city} from a drop-down list")`

Reply:
Done